colors/material.vim

```diff
@@ -414,5 +414,83 @@ if has('nvim')
   let g:terminal_color_13 = g:terminal_color_5
   let g:terminal_color_14 = g:terminal_color_6
   let g:terminal_color_15 = g:terminal_color_7
+elseif has('terminal')
+    if g:material_theme_style == 'palenight'
+        let g:terminal_ansi_colors = [
+                    \ "#292D3E",
```

Comment:
Can we use the colors defined in the colors map so that these colors are defined in one place?

Reply:
Once this gets fixed this we should be able to land this :)